eth/1-security-council-phase-0/README.md

```diff
@@ -0,0 +1,202 @@
+   derivation path is m/44'/60'/0'/0/0. By calling the script with
+   `just simulate 1` it will derive the address using
+   m/44'/60'/1'/0/0 instead.
+
+![](./images/tenderly-overview-network.png)
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+1. There is only a single state override at address
+   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`, which overrides
+   storage slot `0x4` to new value `0x1`. This override is only
+   intended to change the Foundation multisig's quorum threshold to 1
+   so we can perform a tenderly simulation of the execution,
+2. The `ProxyAdmin` contract at
+   `0x543ba4aadbab8f9025686bd03993043599c6fb04`'s `_owner` is changed
+   to a new multisig, and the configuration of this new multisig
+   correctly implements the two approved proposals:
+   1. [Security Council: Vote #1](https://vote.optimism.io/proposals/27439950952007920118525230291344523079212068327713298769307857575418374325849).
+   2. [Security Council Membership Ratification](https://link.to/be/determined).
+3. Both of the other state changes are nonce changes only.
+
+
+All of these addresses should be part of the Optimism Governance vote
+that approves this upgrade if this is a [Normal
+Operation](https://github.com/ethereum-optimism/OPerating-manual/blob/1f42a3766d084864a818b93ce7ba0857a4a846ea/Security%20Council%20Charter%20v0.1.md#normal-operation).
+
+![](./images/tenderly-state-diff.png)
+
+
+#### 3.3. Extract the domain hash and the message hash to approve.
+
+Now that we have verified the transaction performs the right
+operation, we need to extract the domain hash and the message hash to
+approve.
+
+Go back to the "Overview" tab, and find the
+`GnosisSafe.checkSignatures` call. This call's `data` parameter
+contains both the domain hash and the message hash that will show up
+in your Ledger.
+
+Here is an example screenshot. Note that the hash value may be
+different:
+
+![](./images/tenderly-hash.png)
+
+It will be a concatenation of `0x1901`, the domain hash, and the
+message hash: `0x1901[domain hash][message hash]`.
+
+Note down this value. You will need to compare it with the ones
+displayed on the Ledger screen at signing.
+
+### 4. Approve the signature on your ledger
+
+Once the validations are done, it's time to actually sign the
+transaction. Make sure your ledger is still unlocked and run the
+following:
+
+``` shell
+just sign # or just sign <hdPath>
+```
+
+> [!IMPORTANT] This is the most security critical part of the
+> playbook: make sure the domain hash and message hash in the
+> following two places match:
+
+1. on your Ledger screen.
+2. in the Tenderly simulation. You should use the same Tenderly
+   simulation as the one you used to verify the state diffs, instead
+   of opening the new one printed in the console.
+
+There is no need to verify anything printed in the console. There is
+no need to open the new Tenderly simulation link either.
```

Comment:
Comment only, I think not needing to check the terminal output is a good improvement

Reply:
We are in agreement, my comment was just expressing that I like this change. :)